src/rime/dict/dictionary.cc

```diff
@@ -147,9 +148,9 @@ bool DictEntryIterator::Skip(size_t num_entries) {
 // Dictionary members
 
 Dictionary::Dictionary(const string& name,
-                       const an<Table>& table,
-                       const an<Prism>& prism)
-    : name_(name), table_(table), prism_(prism) {
+                       an<Table> table,
+                       an<Prism> prism)
+    : name_(name), table_(std::move(table)), prism_(std::move(prism)) {
```

Comment:
I don't understand how move semantics works here.
reminder: `an<T>` is short for `shared_ptr<T>`

Reply:
This is done automatically by clang-tidy